pkgs/development/python-modules/ihatemoney/default.nix

```diff
@@ -0,0 +1,61 @@
+{ python3Packages, lib, fetchFromGitHub, nixosTests }:
+
+python3Packages.buildPythonPackage rec {
+   pname = "ihatemoney";
+   version = "4.1";
+
+   src = fetchFromGitHub {
+      owner = "spiral-project";
+      repo = pname;
+      rev = version;
+      sha256 = "1ai7v2i2rvswzv21nwyq51fvp8lr2x2cl3n34p11br06kc1pcmin";
+   };
+
+   propagatedBuildInputs = with python3Packages; [
+      alembic
+      aniso8601
+      Babel
+      blinker
+      click
+      dnspython
+      email_validator
+      flask
+      flask-babel
+      flask-cors
+      flask_mail
+      flask_migrate
+      flask-restful
+      flask_script
+      flask_sqlalchemy
+      flask_wtf
+      idna
+      itsdangerous
+      jinja2
+      Mako
+      markupsafe
+      python-dateutil
+      pytz
+      six
+      sqlalchemy
+      werkzeug
+      wtforms
+      psycopg2 # optional, for postgresql support
+   ];
+
+   checkInputs = with python3Packages; [
+      flask_testing
+   ];
+
+   passthru.tests = {
+      inherit (nixosTests) ihatemoney;
+   };
+   meta = {
+      homepage = "https://ihatemoney.org";
+      description = "A simple shared budget manager web application";
```

Comment:
web application so it probably does not belong in `python-packages.nix`

Reply:
I tried to use `buildPythonApplication` but using the `wsgi.py` needs to use this package as a python module and not a python application. Hence the placement in python-packages.nix.